src/postgres/src/backend/commands/tablecmds.c

```diff
@@ -796,9 +796,6 @@ DefineRelation(CreateStmt *stmt, char relkind, Oid ownerId,
 										  InvalidOid,
 										  typaddress);
 
-	/* Store inheritance information for new rel. */
```

Comment:
Use `if !IsYugaByteEnabled() {} ` instead of removing it.

Reply:
So this was a bit strange actually. This extra call was causing a bug, but doesn't seem to have anything to do with Yugabyte integration, it should ideally cause a bug in postgres code too... When I checked their code to see why it wasn't affecting them, I couldn't find this line in the postgres code (11.2) either. I don't know how this extra call landed in the YB code.

Reply:
So this function call was causing a bug that I have explained in the commit message. This should have ideally caused a bug in vanilla postgres code as well, and I couldn't find these lines in vanilla postgres either. Git blame in yugabyte code base didn't give me any clues as to how this line landed in yugabyte code base but not in postgres code?
But this line shouldn't be there, whether Yugabyte enabled or not.

Reply:
@ndeodhar  Do let me know if I'm missing something, have been quite flummoxed how this happened. :)